Sources/SotoCore/Utils/base64.swift

```diff
@@ -70,8 +70,7 @@
 
 // MARK: - Extensions -
 
-public extension String {
-    @inlinable
+extension String {
```

Comment:
This is no longer `@inlinable` - is that deliberate?

Reply:
@inlinable is for public symbols. It basically another module might want to inline this symbol